packaging/nnstreamer.spec

```diff
@@ -455,6 +446,13 @@ cp -r result %{buildroot}%{_datadir}/nnstreamer/unittest/
 %files -n nnstreamer-tizen-internal-capi-devel
 %{_includedir}/nnstreamer/nnstreamer-tizen-internal.h
 
+%if 0%{?enable_nnfw_r}
+%files nnfw
+%manifest nnstreamer.manifest
+%defattr(-,root,root,-)
+%{_prefix}/lib/nnstreamer/filters/libnnstreamer_filter_nnfw.so
```

Comment:
Instead of hard-coded 'lib', it is better to use %{_libdir}.

Reply:
No.
For application/package specific paths, it is /usr/lib/NAME/whatsoever, not %{_libdir}/NAME/whatsoever.